### PR TITLE
Update duplicate meta descriptions

### PR DESCRIPTION
**Pull request by walsehgal**
Update duplicate meta descriptions and increase relevance.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/create-test-tune-dlp-policy.md [hunk 1]
```diff
@@ -16,8 +16,7 @@ ms.collection:
 search.appverid: 
 - MET150
 ms.assetid: 59414438-99f5-488b-975c-5023f2254369
-description: "The easiest, most common way to get started with DLP policies is to use one of the templates included in Office 365.
-"
+description: In this article, you'll learn how to create, test and tune a DLP policy according to yuor organizational needs.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-test-tune-dlp-policy.md]:
```suggestion
description: In this article, you'll learn how to create, test, and tune a DLP policy according to your organizational needs.
```

File: microsoft-365/compliance/compliance-manager-release-notes.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ ms.collection: M365-security-compliance
 search.appverid: 
 - MOE150
 - MET150
-description: "Microsoft Compliance Manager is a free workflow-based risk assessment tool in the Microsoft Service Trust Portal. Compliance Manager enables you to track, assign, and verify regulatory compliance activities related to Microsoft cloud services."
+description: In this article you will find release notes containing information about new features and known issues (to be resolve in upcoming releases) in the Microsoft Complaince Manager.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-release-notes.md]:
```suggestion
description: In this article, you will find release notes containing information about new features and known issues (to be resolve in upcoming releases) in the Microsoft Compliance Manager.
```

File: microsoft-365/compliance/compliance-manager-overview.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ ms.collection: M365-security-compliance
 search.appverid: 
 - MOE150
 - MET150
-description: "Microsoft Compliance Manager is a free workflow-based risk assessment tool in the Microsoft Service Trust Portal. Compliance Manager enables you to track, assign, and verify regulatory compliance activities related to Microsoft cloud services."
+description: In this article you will get an overview of the Microsoft Compliance Manager; a free workflow-based risk assessment tool in the Microsoft Service Trust Portal.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-overview.md]:
```suggestion
description: In this article, you will get an overview of the Microsoft Compliance Manager; a free workflow-based risk assessment tool in the Microsoft Service Trust Portal.
```

File: microsoft-365/compliance/data-investigations-release-notes.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ search.appverid:
 - MET150
 ms.assetid: 
 
-description: "This article describes the new Data Investigations (Preview) tool in Microsoft 365."
+description: In this article you will find release notes the Data Investigations (Preview) tool in Microsoft 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-investigations-release-notes.md]:
```suggestion
description: In this article, you will find release notes for the Data Investigations (Preview) tool in Microsoft 365.
```

File: microsoft-365/compliance/gdpr-dsr-Dynamics365.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Dynamics 365 Data Subject Requests for the GDPR and CCPA
-description: Guide for how to use Microsoft products, services, and administrative tools to help our controller customers find and act on personal data to respond to DSR and CCPA requests.
+description: This guide will help you understand how to find and act on personal data and respond to DSR and CCPA request made by Dynamics 365 customers.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-dsr-Dynamics365.md]:
```suggestion
description: This guide will help you understand how to find and act on personal data and respond to DSR and CCPA requests made by Dynamics 365 customers.
```

File: microsoft-365/compliance/gdpr-dsr-Intune.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Intune Data Subject Requests for the GDPR and CCPA
-description: Guide for how to use Microsoft products, services, and administrative tools to help our controller customers find and act on personal data to respond to DSR and CCPA requests.
+description: This guide will help our controller customers understand on how to find and act on personal data and respond to DSR and CCPA requests by customers using Microsoft Intune.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-dsr-Intune.md]:
```suggestion
description: This guide will help our controller customers understand how to find and act on personal data and respond to DSR and CCPA requests by customers using Microsoft Intune.
```

File: microsoft-365/compliance/office-365-customer-managed-encryption-features.md [hunk 7]
```diff
@@ -12,7 +12,7 @@ localization_priority: None
 search.appverid:
 - MET150
 ms.collection: Strat_O365_Enterprise
-description: "Summary: Understand data resiliency in Microsoft Office 365."
+description: In this article, you'll learn about encryption technologies that you can manage and configure in Microsoft Office 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-customer-managed-encryption-features.md]:
```suggestion
description: In this article, you will learn about encryption technologies that you can manage and configure in Microsoft Office 365.
```

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ ms.collection:
 - Strat_O365_Enterprise
 - M365-security-compliance
 - Strat_O365_Enterprise
-description: "Summary: Understand data resiliency in Microsoft Office 365."
+description: In this article you'll learn about risks to the Office-365 service and the encryption technologies available for protection.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-risks-and-protections.md]:
```suggestion
description: In this article, you'll learn about risks to the Office-365 service and the encryption technologies available for protection.
```

File: microsoft-365/compliance/office-365-service-encryption.md [hunk 9]
```diff
@@ -13,7 +13,7 @@ localization_priority: None
 search.appverid:
 - MET150
 ms.collection: Strat_O365_Enterprise
-description: "Summary: Understand data resiliency in Microsoft Office 365."
+description: In this article, you'll learn about Service Encryption that in addition to volume-level encrpytion can be used to encrypt customer data.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-service-encryption.md]:
```suggestion
description: In this article, you'll learn about Service Encryption that, in addition to volume-level encryption, can be used to encrypt customer data.
```

File: microsoft-365/compliance/overview-data-investigations.md [hunk 10]
```diff
@@ -16,7 +16,7 @@ search.appverid:
 - MET150
 ms.assetid: 
 
-description: "This article describes the new Data Investigations (Preview) tool in Microsoft 365."
+description: In this article, you'll learn about the Data Investigations (Preview) tool in Microsoft 365. Data Investigations tool helps you assess and remediate the data spillage.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/overview-data-investigations.md]:
```suggestion
description: In this article, you'll learn about the Data Investigations (Preview) tool in Microsoft 365. The Data Investigations tool helps you assess and remediate the data spillage.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@walsehgal - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by walsehgal** [comment]
@DaniHalfin

**Pull request merged**